exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook_requirements.txt

```diff
@@ -7,4 +7,4 @@ jupysql==0.10.10
 sqlalchemy_exasol==4.6.3
 stopwatch.py==2.0.1
 boto3==1.26.163
-exasol-notebook-connector @ git+https://github.com/exasol/notebook-connector.git@main
+exasol-notebook-connector @ git+https://github.com/exasol/notebook-connector.git@bug/tmp_fix_deps
```

Comment:
Do you plan to stay on this branch?

Reply:
The other PRs in sagemaker-extension and transformer-extensions are now merged. I reverted this.